avocado/plugins/multiplex.py

```diff
@@ -120,6 +123,10 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_ALL_OK)
 
         variants = multiplexer.MuxTree(mux_tree)
+        if args.get_count:
+            log.info("%d" % len(list(enumerate(variants))))
```

Comment:
(I spent some time trying to find the fastest implementation as it's always calculated in `avocado.core.multiplexer.Mux.get_number_of_tests`)